docs/generator.go

```diff
@@ -0,0 +1,685 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
+
+func main() {
+	contents, err := ioutil.ReadFile("../schema.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var schema SchemaData
+	if err := json.Unmarshal(contents, &schema); err != nil {
+		log.Fatal(err)
+	}
+
+	
+	fmt.Println(schema.Title)
+
+	tpl, err := template.ParseFiles("doc.md")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	//if err = tpl.Execute(os.Stdout, schema); err != nil {
+	//	log.Fatal(err)
+	//}
```

Comment:
Commented code is dead code! Should be removed unless there's a documentation reason for it to stay there :)